modules/claims_api/app/swagger/claims_api/form_2122_model_swagger.rb

```diff
@@ -19,19 +19,19 @@ class Form2122ModelSwagger
         key :description, 'Required by JSON API standard'
         key :required, %i[poa_code poa_first_name poa_last_name]
 
-        property :poa_code do
+        property :poaCode do
```

Comment:
Why change it to camel case instead of underscore?

Reply:
as per our offline convo. This change was because the docs didnt match the schema. The schema uses camelCase.